packages/agoric-cli/src/deploy.js

```diff
@@ -1,17 +1,23 @@
-/* global process setTimeout setInterval clearInterval */
+/* eslint-env node */
```

Comment:
Spread the word.

Reply:
hm. I kinda like the explicit list.

Reply:
Oh, indeed. I must have been cross-eyed when I wrote this because I thought that the environment might provide a better hint than `any` types for all these globals to TypeScript, but this is an _eslint_ pragma so it probably does not.